From: Paul Tagliamonte <paultag@ubuntu.com>
Author: Daniel Diaz <dydyam@gmail.com>
Date: Thu, August 12th, 2010  07:06:01  +0000
Subject: Keybinding fix: Change `xterm' to `x-terminal-emulator'
Description:
 This changes the default keybinding from xterm to x-terminal-emulator,
 which is more complient with Debian policy. This is a local change only,
 for the most part.
Origin: local
Forwarded: not-needed

diff --git a/data/keys b/data/keys
index 7e2557c..b02f80f 100644
--- a/data/keys
+++ b/data/keys
@@ -58,7 +58,7 @@ Mod4 8 :Tab 8
 Mod4 9 :Tab 9
 
 # open a terminal
-Mod1 F1 :Exec xterm
+Mod1 F1 :Exec x-terminal-emulator
 
 # open a dialog to run programs
 Mod1 F2 :Exec fbrun
